storage/postgres/notificator.go

```diff
@@ -323,40 +365,31 @@ func (n *Notificator) getRecipients(platformID string) []*types.Platform {
 func (n *Notificator) sendNotificationToPlatformConsumers(platformConsumers []storage.NotificationQueue, notification *types.Notification) {
 	for _, consumer := range platformConsumers {
 		if err := consumer.Enqueue(notification); err != nil {
-			log.C(n.ctx).WithError(err).Infof("consumer %s notification queue returned error %v", consumer.ID(), err)
+			log.C(n.ctx).WithError(err).Infof("Consumer %s notification queue returned error %v", consumer.ID(), err)
 			consumer.Close()
 		}
 	}
 }
 
 func (n *Notificator) stopConnection() {
-	err := n.stopListening()
-	logger := log.C(n.ctx)
-	if err != nil {
-		logger.WithError(err).Info("could not unlisten notification channel")
-	}
+	n.stopProcessing()
```

Comment:
write godoc that you expect this to be caught in `processNotifications` which would in turn close all consumers and unlisten